tests/scripts/release-post.test.js

```diff
@@ -194,13 +194,13 @@ test('fetchVersionPolicy(<version>)', (t) => {
   const changelogFixture = path.resolve(__dirname, 'CHANGELOG.fixture.md')
   const changelogLegacyFixture = path.resolve(__dirname, 'CHANGELOG.fixture.legacy.md')
 
-  t.test('finds "Stable" version policy', (t) => {
+  t.test('finds "Current" version policy', (t) => {
     const github = nock('https://raw.githubusercontent.com')
       .get('/nodejs/node/v4.1.0/CHANGELOG.md')
       .replyWithFile(200, changelogFixture)
 
     releasePost.fetchVersionPolicy('4.1.0').then((policy) => {
-      t.equal(policy, 'Stable')
+      t.equal(policy, 'Current')
```

Comment:
> I vote yes.

I would agree in the eyes of consistency, but that would possibly mean updating all previous changelogs/blogs etc, I'm not sure that's preferable.

@fhemberger as I'm not sure what to do about previous releases, I've made two branches with contains fixes for both scenarios:

- `rename-stable-old-still-stable`: old releases should still be named "Stable"
- `rename-stable-incl-old-releases`: rename old releases to "Stable" as well

Merge or cherry-pick what ever commit you see fit. IMO keeping "Stable" for old releases would be okey for this PR, further discussion and a permanent fix could be raised in a separate issue/PR.